cucumber/authenticators_jwt/features/authn_jwt_audience.feature

```diff
@@ -0,0 +1,68 @@
+Feature: JWT Authenticator - Check audience
+
+  Verify the authenticator works correctly audience variable and aud claim
+
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: token-app-property
+
+      - !group hosts
+
+      - !permit
+        role: !group hosts
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt "token-app-property" variable to value "host"
+
+  @sanity
+  Scenario Outline: Audience tests
+    Given I extend the policy with:
+    """
+    - !variable conjur/authn-jwt/raw/audience
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: valid-project-id
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I successfully set authn-jwt "audience" variable to value "<audience>"
+    And I issue a JWT token:
+    """
+    {
+      "project-id":"valid-project-id",
+      "host":"myapp"
+      <aud>
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is <http_code>
+    And The following appears in the log after my savepoint:
+    """
+    <log>
+    """
+    Examples:
+      | Test       | audience        | aud                                          | http_code | log                                                                                                                                       |
```

Comment:
Can you move the `,` to the token instead of the inline

Reply:
Done